cartographer/mapping_2d/sparse_pose_graph.h

```diff
@@ -106,9 +106,9 @@ class SparsePoseGraph : public mapping::SparsePoseGraph {
                                 const transform::Rigid3d& pose,
                                 const common::Time& time) override
       EXCLUDES(mutex_);
-  transform::Rigid3d GetClosestTrajectoryPose(int trajectory_id,
-                                              const common::Time& time)
-      EXCLUDES(mutex_);
+  transform::Rigid3d GetInterpolatedGlobalTrajectoryPose(
+      int trajectory_id,
```

Comment:
const int?

Reply:
It's not important for a caller of this member function to know that we don't change the integer inside.